docs/reference/supported-environments.md

```diff
@@ -47,17 +47,25 @@ We recommend running Camunda 8 Self-Managed in a Kubernetes environment. We prov
 
 ### Deployment options
 
-With the right configuration, Camunda 8 Self-Managed can be deployed on any [Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/#benefits) distribution (cloud or on-premise). However, we officially test and support a specific list of platforms.
+Camunda 8 self-managed supports the following [CNCF Certified Kubernetes distributions](https://www.cncf.io/training/certification/software-conformance/#benefits). Some distributions include custom patches or upstream source code changes. Camunda will not provide support for issues arising from these changes. The management of the Kubernetes cluster as well as underlying infrastructure, remains your responsibility; we do not offer direct support for these components.
```

Comment:
why this paragraph is not moved to `Kuberentes` block below? it only applies to Kubernetes flavor of installation and `Deployment options` in general